seeds/dex/trades/dex_trades_seed.csv

```diff
@@ -379,4 +379,10 @@ arbitrum,camelot,1,2023-02-10,0x03bad3b7a62d5dcde805ebcf1441e6966fdf66f4d03fabc0
 arbitrum,camelot,1,2023-02-06,0x04fd14aa1471a9561a66fd804a5c995ec211aa7e96bbcbeb48801efdbdb236d6,3,0x82af49447d8a07e3bd95bd0d56f35241523fbab1,0.22457814589748834,0x088cd8f5ef3652623c22d48b1605dcfe860cd704,
 arbitrum,camelot,1,2023-02-16,0x0aba2b8e2046b670abd3046e28e55e5fb521b6271d2ec8fb72c54df1f5977079,3,0x82af49447d8a07e3bd95bd0d56f35241523fbab1,0.707795125477355,0x51318b7d00db7acc4026c88c3952b66278b6a67f,1759.9819725646116
 arbitrum,camelot,1,2023-02-08,0x317879e802e220cd2b910c2b477406506eff5ef62894344adc9f7e26fa3589e0,3,0xff970a61a04b1ca14834a43f5de4533ebddb5cc8,466.224209,0xba42a714bc9c16be6a1ce5eb66ab4deb3904862a,
-arbitrum,camelot,1,2023-02-20,0x2d820ba0c8ee0b2b3dfbe1002520bfd952fd65e6f52217722fdd83ed44b982cb,3,0x82af49447d8a07e3bd95bd0d56f35241523fbab1,0.18770390514247653,0x1426cf37caa89628c4da2864e40cf75e6d66ac6b,225
\ No newline at end of file
+arbitrum,camelot,1,2023-02-20,0x2d820ba0c8ee0b2b3dfbe1002520bfd952fd65e6f52217722fdd83ed44b982cb,3,0x82af49447d8a07e3bd95bd0d56f35241523fbab1,0.18770390514247653,0x1426cf37caa89628c4da2864e40cf75e6d66ac6b,225
+ethereum,xchange,1,2023-06-05,0x46da856ff5390f122260b3ddeff8c84e6471d715722bcec69410859a9dd7a9ce,26,0x70008f18fc58928dce982b0a69c2c21ff80dca54,6177.083844891557065659,0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2,0.25809555
+ethereum,xchange,1,2023-06-04,0xb9cf9af8c09ee4b01f7998e4d09b5732e3d59db577453d8f649c3368a5be4723,3,0x7102dc82ef61bfb0410b1b1bf8ea74575bf0a105,16740.414870869491102082,0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2,349.842891200744390656
+ethereum,xchange,1,2023-06-04,0xdca9e79f3d41644b579994c0e62f73e8741605b85bb9a80fcc6d173180fd891c,26,0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2,264.386114548212846651,0x7103ebdbf1f89be2d53eff9b3cf996c9e775c105,16740.857191421503408067
+ethereum,xchange,1,2023-06-12,0xf71d8fd364e2402eb11ba7bfbd9b59336e2b075571c1523d119f74a11a1986e3,170,0x117546d1467d80c6bde13910412c724383260cf9,345579.7727468311529198,0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2,0.1
+ethereum,xchange,1,2023-06-12,0x7da422240ab88976df8c9694114daf8b8cfc335e1f851a8c175bca99e5e4cdcd,200,0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2,0.134508790612232095,0x117546d1467d80c6bde13910412c724383260cf9,477750.8030874053723140
+ethereum,xchange,1,2023-06-12,0x4eba2bbed2a473a3a6b2fbfc1d784940b5157afc285c35aff1200e491ea1346a,115,0xdbc7945c5403c589ec39a9aa8c5af234c706f6a2,14049529527.028758241707276441,0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2,0.1
```

Comment:
Hi @mike-x7f,
seems currently this file only contains seed records for the `ethereum` chain.
Do you want to add xchange seeds for other chains as well? 
Like for [bnb, arbitrum, polygon] chains.

Reply:
I could add some but that test requires both the bought and sold tokens to be in the tokens.erc20 table in order to get the bought and sold amounts. Xchange is a new DEX and doesn't have a lot of volume on those chains yet, and all of the trades so far involve at least one token that isn't in tokens.erc20 yet. I am adding tokens to tokens.erc20 in this PR so I can use check_dex_seed, but I think it will fail until those tokens are added. I ran a manual test on those chains that you can find in the link in my other comment. All of the fields are populated as expected except for those that require tokens.erc20 data (symbol, pair, bought/sold amount).